validation/resource-config_test.go

```diff
@@ -0,0 +1,93 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func TestResourceConfig(t *testing.T) {
+	isExcluded := map[uint64]bool{
+		888:          true,
+		997:          true,
+		11155421:     true,
+		11763071:     true,
+		129831238013: true,
```

Comment:
These chains to not specify the address of the `SystemConfigProxy`. So I excluded them.